diff -aurN gtkglext-1.0.6.orig/configure.in gtkglext-1.0.6/configure.in
--- gtkglext-1.0.6.orig/configure.in	2004-03-10 10:47:37.000000000 +0100
+++ gtkglext-1.0.6/configure.in	2006-02-19 10:20:47.810661176 +0100
@@ -63,6 +63,10 @@
 m4_define([pangoft2_pkg], [pangoft2])
 m4_define([pangoft2_required_version], [1.0.0])
 
+# PangoX
+m4_define([pangox_pkg], [pangox])
+m4_define([pangox_required_version], [1.0.0])
+
 # GDK
 m4_define([gdk_pkg], [gdk-2.0])
 m4_define([gdk_required_version], [2.0.0])
@@ -789,7 +793,7 @@
 # CFLAGS and LIBS
 ##################################################
 
-GDKGLEXT_PACKAGES="gdk_pkg pango_pkg gmodule_pkg"
+GDKGLEXT_PACKAGES="gdk_pkg pango_pkg pangox_pkg gmodule_pkg"
 GDKGLEXT_EXTRA_CFLAGS="$GL_CFLAGS $GDKGLEXT_WIN_CFLAGS"
 GDKGLEXT_EXTRA_LIBS="$GL_LIBS $GDKGLEXT_WIN_LIBS"
 GDKGLEXT_DEP_CFLAGS="$GDKGLEXT_EXTRA_CFLAGS `$PKG_CONFIG --cflags $GDKGLEXT_PACKAGES`"
